ironfish/src/network/peerNetwork.ts

```diff
@@ -899,6 +898,14 @@ export class PeerNetwork {
         continue
       }
 
+      // Recently evicted means the transaction is relatively low feeRate
+      // mempool + wallet have already processed it. It could stil be download
+      // and forward to other peers but since it is a low feeRate and
+      // other peer mempools are likely at capacity, just drop it
+      if (this.node.memPool.recentlyEvicted(hash)) {
```

Comment:
I hadn't thought of this before, but I think this kind of assumes that most peer mempools will be the same size -- if you assume that every peer will have drastically different mempool sizes (like your node has 3MB mempool and peers have 100MB), is this still useful to check?

Reply:
Yea agreed this could mean that nodes with smaller mempools forward fewer transactions. I increased the min maxMempoolSizeBytes to 20Mb which should alleviate some of this. Also at this point we have already broadcasted it once to all the peers we had when it was originally added to the mempool so this would just be downloading to potentially broadcast it to newly added peers. Given that it is also evicted value of downloading doesn't seem worth it. I'll update this comment to also include the fact that we've already broadcasted it once